.vscode/launch.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Accidental addition?

Reply:
No. This makes it possible to run the unit test and debug them very easily in VSCode. I thought other people might enjoy it. Should work without any change on anyone's computer.

I would also completely understand if you would rather keep the repo IDE-agnostic ;)